README.md

```diff
@@ -2,14 +2,26 @@
 
 # VSCode Extension: Status Bar Env Var
 
-Display system environment variables that VSCode is running in the context of in the status bar.
+Display system environment variables in the VSCode status bar and overwrite them easily.
+
+## Limitations
 
 This extension only runs locally and not in remote environments. This means that when using a remote environment the environment variables displayed are those of the local machine running the VSCode UI and not those of the remote VSCode Server.
 
+Overwritten environmental variables are only applied to shell environmental variables, not the VSCode built in ${env} variable.
```

Comment:
I think it would be better if the env var applied to VSCode. That's largely what the extension displays now, what is applied to VSCode, not just the shell. This way it is possible to do some things like, see what DOCKER_HOST VSCode has been configured for which will influence the docker host docker extensions will connect to. What's the motivation for only applying them to the shell?

Reply:
I wanted it to apply to VSCode, specifically tasks and launch configs, but it seems like that doesn't work due to a limitation in VS Code. See https://github.com/microsoft/vscode/issues/142826.

Note that scripts launched from a task or launch config can still access the updated env var, it is just the ${env} variable that is not updated. I agree this sucks because it makes it a whole less flexible.
A workaround could be explicitly modifying the Env config of each task and launch config.